src/adapter/src/coord/sequencer/inner/create_materialized_view.rs

```diff
@@ -337,27 +337,23 @@ impl Coordinator {
             role_metadata: session.role_metadata().clone(),
         };
 
-        // Acquire read holds at all the REFRESH AT times.
-        // Note that we already acquired a possibly non-precise read hold at mz_now() in the purification,
-        // if any of the REFRESH options involve mz_now(). But now we can acquire precise read holds, because by now
-        // the REFRESH AT expressions have been evaluated, so we can handle something like
-        // `mz_now()::text::int8 + 10000`;
+        // Check whether we can read all inputs at all the REFRESH AT times.
```

Comment:
Maybe you could also check whether the bundle of IDs in the read hold covers all the IDs that we do need for this MV?

Reply:
Done. I'm throwing `AdapterError::ChangedPlan`, is that ok?

Reply:
Sounds alright, but I'm not the expert here